### PR TITLE
Scenario objects show on gridview

### PR DESCRIPTION
**Pull request by ningning621**
- includes simulator files that we will hopefully understand how to use in the future

### CODE REVIEW

File: basestation/simulator/simulator.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+"""
+Class that contains simulator related stuff
+"""
+
+class Simulator:
+    def __init__(self):
+        self.listOfScenario = []
+
+    def set_scenario_list(self, list):
+        self.listOfScenario = list
```
Comment (yangdanny97) [inline basestation/simulator/simulator.py]: you're going to need comments

File: basestation/simulator/simulator_manager.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from basestation.simulator.simulator import Simulator
+
+"""
+Manages the simulator
```
Comment (yangdanny97) [inline basestation/simulator/simulator_manager.py]: why is this necessary again

File: gui/main.py [hunk 3]
```diff
@@ -55,6 +56,26 @@ def make_app(self):
...
+    """
+    Adds scenario objects to gridview
+    """
+    def post(self):
+        listofscenario = json.loads(self.request.body.decode())
+        # BaseStation().get_simulator_manager().simulator.set_scenario_list(listofscenario)
+        counter = 0
+        for object in listofscenario:
+            print(object)
+            BaseStation().vision_manager.update_location("Object " + str(counter), {
```
Comment (yangdanny97) [inline gui/main.py]: if possible differentiate naming of obstacles and bots

Comment (yangdanny97) [reply gui/main.py]: also, you should clear the existing scenario first

**Review by yangdanny97 — COMMENTED** [review]
i'm going to assume the frontend of the scenarios section doesn't have any issues

File: basestation/simulator/simulator_manager.py (hunk 2 above)
Comment (ningning621) [reply basestation/simulator/simulator_manager.py]: good question

**Review by yangdanny97 — APPROVED** [review]
looks okay to me, i assume that this has been tested already if not then please test
